src/wallet/puzzles/chialisp_deserialisation.clvm

```diff
@@ -0,0 +1,64 @@
+(mod (input)
+
+  (defconstant CONS_BOX_MARKER 0xFF)
+
+  (defconstant MAX_SINGLE_BYTE 0x7F)
+  (defconstant MAX_TWO_BYTE 0xbf)
+  (defconstant MAX_THREE_BYTE 0xdf)
+  ; Constants (FOUR,FIVE,SIX) are unused, but intentionally left to aid
+  ; understanding of the serialization format
```

Comment:
if we would support binary integral constants it would be even nicer! :)
`0b10111111` and `0b11011111`